src/main/groovy/io/seqera/wave/service/aws/S3ClientFactory.groovy

```diff
@@ -59,4 +64,13 @@ class S3ClientFactory {
         log.info("Creating S3 client with configuration: $builder")
         return builder.build()
     }
+
+    @Singleton
+    @Named('awsS3Client')
```

Comment:
```suggestion
    @Named('BuildLogsS3Client')
```

Reply:
This S3 client can be used in other scenarios too because its using default aws region

Reply:
But currently is only used for build logs right?

Reply:
But ok, let's `DefaultS3Client` then

Reply:
done